tests/testthat/test-methods.R

```diff
@@ -31,6 +31,7 @@ test_that("as.data.frame.lints", {
     ),
     "lint"
   )
+  expect_type(l1, "list")
```

Comment:
Nit: Add a similar new test for `lint()`?

Reply:
thanks for calling it out... i didn't see a "nice" place to add it on a quick glance. but i'll have another go.